src/components/PresentationContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { createContext, PropsWithChildren, useContext, useMemo } from "react";
+import { GridStyle } from "src/components/Table";
+import { LabelSuffixStyle } from "src/forms/FormContext";
+
+export type PresentationContextProps = {
+  // Value will be ignored if passed. The value is set based on existence of nested PresentationProviders
+  baseContext?: boolean;
+  numberAlignment?: "left" | "right";
```

Comment:
Ah yeah...that's interesting. I guess I was thinking we'd have only a few...and may not be just things from `BeamTextFieldProps`... I guess `fieldProps` I kinda like in terms of nesting, although I might hand-code it instead of picking it from the others. Not sure why. But yeah.